### PR TITLE
update ember-cli

### PR DESCRIPTION
**Pull request by efx**
Ran `ember-cli-updater` in the root so I could try fixing #72. Wanted to open this PR separately.

### CODE REVIEW

File: config/ember-try.js [hunk 1]
```diff
@@ -10,18 +10,6 @@ module.exports = function() {
   ]).then((urls) => {
     return {
       scenarios: [
-        {
-          name: 'ember-lts-2.16',
```
Comment (offirgolan) [inline config/ember-try.js]: Can you remove the scenario from the travis config?

Comment (efx) [reply config/ember-try.js]: certainly! thanks for catching it @offirgolan.

**Comment by efx** [comment]
Looks like one failure is from babel@5 in ember-cli-changelog. I can try bumping that as well.

**Comment by offirgolan** [comment]
Yeah, give it a go and lets see if that fixes it. If not I'll see if I resolve it in another PR

**Comment by efx** [comment]
@offirgolan so turns out the beta versions are not available via yarn's package repositories: https://yarnpkg.com/en/package/ember-cli-changelog. I'm asking in discourse if anyone's seen this kind of error from babel 5 + ember-cli + node 6. Another route would be seeing if we can get the beta versions pushed to yarn's repositories or however they can be made available with `yarn install`.

**Comment by offirgolan** [comment]
@efx you should be able to target it by replacing the current version with `html-next/ember-cli-changelog#v0.4.0-beta.1` in the package.json.

**Pull request merged**